sdk/core/amqp-common/ci.yml

```diff
@@ -9,9 +9,9 @@ pr:
       - master
   paths:
     include:
-      - sdk/core/amqp-common/
+      - sdk/core/
 
 jobs:
   - template: ../../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
     parameters:
-      PackageName: "@azure/amqp-common"
+      ServiceDirectory: sdk/core/
```

Comment:
Is it necessary to have a trailing slash? Also it would be nice if we could avoid the duplication of the "sdk" like we did in the .NET repo.

Reply:
Fixed.